tests/acceptance/test_organization_events_v2.py

```diff
@@ -3,15 +3,114 @@
 import pytz
 from mock import patch
 
+from six.moves.urllib.parse import urlencode
+
 from sentry.testutils import AcceptanceTestCase, SnubaTestCase
 from sentry.utils.samples import load_data
 from sentry.testutils.helpers.datetime import iso_format, before_now
 
 
-FEATURE_NAMES = "organizations:events-v2"
+FEATURE_NAMES = ["organizations:events-v2", "organizations:transaction-events"]
+
+all_events_query = urlencode(
+    {
+        "sort": ["-timestamp"],
+        "field": ["title", "event.type", "project", "user", "timestamp"],
+        "tag": ["event.type", "release", "project.name", "user.email", "user.ip", "environment"],
+        "name": ["All Events"],
+        "fieldnames": ["title", "type", "project", "user", "time"],
+    },
+    doseq=True,
+)
+
+errors_query = urlencode(
+    {
+        "sort": ["-last_seen", "-title"],
+        "name": ["Errors"],
+        "fieldnames": ["error", "events", "users", "project", "last seen"],
+        "field": ["title", "count(id)", "count_unique(user)", "project", "last_seen"],
+        "tag": ["error.type", "project.name"],
+        "query": ["event.type:error"],
+    },
+    doseq=True,
+)
+transactions_query = urlencode(
+    {
+        "sort": ["-count"],
+        "name": ["Transactions"],
+        "fieldnames": ["transaction", "project", "volume"],
+        "field": ["transaction", "project", "count()"],
+        "tag": ["release", "project.name", "user.email", "user.ip", "environment"],
+        "statsPeriod": ["14d"],
+        "query": ["event.type:transaction"],
+    },
+    doseq=True,
+)
+
+
+def generate_transaction():
+    from datetime import timedelta
+
+    event_data = load_data("transaction")
+
+    # set timestamps
+
+    start_datetime = before_now(minutes=1)
+    end_datetime = start_datetime + timedelta(milliseconds=500)
+    span_end_datetime = start_datetime + timedelta(milliseconds=250)
+
+    def generate_timestamp(date_time):
+        import time
+
+        return time.mktime(date_time.utctimetuple()) + date_time.microsecond / 1e6
+
+    event_data["breadcrumbs"]["values"][0]["timestamp"] = generate_timestamp(start_datetime)
```

Comment:
Do transaction events use breadcrumbs?

Reply:
@markstory They're not consumed in the span interface.

I can't remember why I added this; but my guess was that I saw it in the transaction sample file and updated thee timestamp accordingly https://github.com/getsentry/sentry/blob/bce57cd45dd5e4e96d1e7c115ef027dab41aaa0d/src/sentry/data/samples/transaction.json#L50-L60

I've since removed this line.